Doc/faq/general.rst

```diff
@@ -125,9 +125,9 @@ find packages of interest to you.
 How does the Python version numbering scheme work?
 --------------------------------------------------
 
-Python versions are numbered A.B.C or A.B.  A is the major version number -- it
-is only incremented for really major changes in the language.  B is the minor
-version number, incremented for less earth-shattering changes.  C is the
+Python versions are numbered "A.B.C" or "A.B".  *A* is the major version number -- it
+is only incremented for really major changes in the language.  *B* is the minor
+version number, incremented for less earth-shattering changes.  *C* is the
 micro-level -- it is incremented for each bugfix release.  See :pep:`6` for more
 information about bugfix releases.
```

Comment:
```suggestion
Python versions are numbered "A.B.C" or "A.B":
* *A* is the major version number --- it is only incremented for
  really major changes in the language.
* *B* is the minor version number, incremented for less
  earth-shattering changes.
* *C* is the micro-level --- it is incremented for each bugfix release.

See :pep:`6` for more information about bugfix releases.
```
A few comments:
* Turning this into a list would also make it more readable (see suggestion).
* Using ` ``...`` `  instead of `"..."` and `*...*` might be better, since it stands out more, but it's not essential.
* Using `` :samp:`...` `` for `` :samp:`A.B.C` `` is another option, but not particularly useful in this context.

Reply:
I assume you mean

`` :smap:`{A}.{B}.{C}` ``, no? Otherwise, why use `:samp:` instead of just a literal, italic, etc?